front/pages/api/w/[wId]/workspace-analytics.ts

```diff
@@ -63,6 +63,8 @@ async function getAnalytics(
 ): Promise<GetWorkspaceAnalyticsResponse> {
   const replicaDb = getFrontReplicaDbConnection();
 
+  // TODO(workspace_filtering):  Review needed
```

Comment:
We could maybe get the workspace modelId by passing the auth here if possible to simplify the logic.

Otherwise looks good